pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -462,6 +462,18 @@ func TestEnoughRequests(t *testing.T) {
 				},
 			},
 		},
+		{
+			pod: newResourcePod(
+				framework.Resource{
+					MilliCPU: 1,
+					Memory:   1,
+					ScalarResources: map[v1.ResourceName]int64{
+						kubernetesIOResourceA: 0,
+					}}),
+			nodeInfo:                  framework.NewNodeInfo(newResourcePod(framework.Resource{MilliCPU: 0, Memory: 0})),
```

Comment:
Can we cover this case with `(nodeInfo.Allocatable.ScalarResources[rName] - nodeInfo.Requested.ScalarResources[rName]) < 0`, which mocks the bug we met.

Reply:
Sure. updated the case as `allocatable` 5 and `Requested` 6 for `extendedResourceA`, which should hit the bug we met.